### DISCUSSION TITLE
Add note word count

### DISCUSSION THREAD

**Discussion by robin-mclain** [standardnotes/forum#3344] [Suggestions] [answered]
**Is your feature request related to a problem? Please describe.**
When I'm doing certain types of writing, e.g. creative writing or my morning journal, I set a target for myself to hit a certain word count. I'm used to being able to view that somewhere inside the writing app that I'm working in, but I can't do that in Standard Notes.

**Describe the solution you'd like**
I would love to be able to see the word count for the current note I'm working on somewhere. In the "..." is fine, but it would be way better if I could see it as I'm writing.

**Describe alternatives you've considered**
I found an extension called "action-bar" that is a pain to set up and doesn't work consistently. It also requires me to have a local server running, which is serious overkill for what I'm trying to do. I can find the word count by copy-pasting the text of my note somewhere else, but that means leaving the app to do something that I would expect to be able to do *inside* the app. It's really inconvenient.

**Comment by effieeee** [comment, marked as answer]
Hi @tjmclain, there is also a character/word/paragraph counter for plain text, but I'm afraid we've yet to adapt this counter for other note types like Super and Rich Text 🙏

![image](https://github.com/standardnotes/forum/assets/54989748/a6573f1f-a751-471c-a587-db03cc0f49f9)

**Reply by robin-mclain** [reply]
Ah, thanks for letting me know! I mostly work in super notes so that explains why I wasn't seeing that. I can definitely switch to a plaintext note type for the times I do want to see the word count. Is adding that info for super notes on the roadmap? My problem is solved now, but it would be cool to have that!
